src/extension.ts

```diff
@@ -68,7 +69,7 @@ async function createLanguageClient(): Promise<LanguageClient | undefined> {
     };
 
     return new LanguageClient(
-        "par_language_server",
+        "par",
```

Comment:
The language client uses this ID to determine the config section to use (i.e., the previous value would make the config option be spelled `par_language_server.trace.server`)